pipeline/inputs/opentelemetry.md

```diff
@@ -0,0 +1,51 @@
+---
+description: An input plugin to ingest OTLP Logs, Metrics, and Traces 
+---
+
+# OpenTelemetry
+
+The OpenTelemetry plugin allows you to ingest telemetry data as per the OTLP specification, from various OpenTelemetry exporters, the OpenTelemetry Collector, or Fluent Bit's OpenTelemetry output plugin. 
+
+## Configuration <a href="#configuration" id="configuration"></a>
+
+| Key           | Description                                                                                                                                    | default |
+| ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| host              | The address to listen on                                                                                                                       | 0.0.0.0 |
+| port              | The port for Fluent Bit to listen on                                                                                                           | 4318    |
+| tag_key           | Specify the key name to overwrite a tag. If set, the tag will be overwritten by a value of the key.                                            |         |
+| buffer_max_size   | Specify the maximum buffer size in KB to receive a JSON message.                                                                               | 4M      |
+| buffer_chunk_size | This sets the chunk size for incoming incoming JSON messages. These chunks are then stored/managed in the space available by buffer_max_size.  | 512K    |
+|successful_response_code | It allows to set successful response code. `200`, `201` and `204` are supported.| 201 |
+
+## Getting started
+
+The OpenTelemetry plugin currently supports the following telemetry data types:
+
+|     Type    |    HTTP/JSON  |   HTTP/Protobuf |
+| ----------- | ------------- | --------------- |
+|    Logs     |     Stable    |      Stable     |
+|    Metrics  | Unimplemented |      Stable     |
+|    Traces   |      Beta     |   Unimplemented |
+
+
+Important note: We only support raw traces as of v1.9, which means that any data forwarded to the traces endpoint (`/v1/traces`) will be packed and forwarded, and will NOT be processed by Fluent Bit.
```

Comment:
Considering that we will ship this with 2.0 only. Is this a valid line?

Reply:
Nope I will update it